shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
+            'actionUpdateQuantity',
```

Comment:
You need only the first **updateQuantity** hook. On PrestaShop 1.5+, an alias exists in order to link the new and old name (See table hook_alias)